docs/skopeo.1.md

```diff
@@ -192,6 +192,48 @@ Verify a signature using local files, digest will be printed on success.
 
 **Note:** If you do use this, make sure that the image can not be changed at the source location between the times of its verification and use.
 
+## skopeo sync
+**skopeo copy** [**--source** _source_] [**--source-file** _source-file_] _destination_
+
+Copy all the images from _source_ (or _source-file_) to _destination_.
+
+Useful to keep in sync a local docker registry mirror. It can be used to populate also registries running inside of air-gapped environments.
```

Comment:
Suggest:
Useful to keep in sync with a local docker registry mirror. It can also be used to populate registries running inside of air-gapped environments.